Duke.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class Duke {
+
+    private static int Count = 0;
+    private static Task[] List = new Task[10]; //creating list of Task for inputs
+
+    //add task object into list
+    public static void addList(Task s){
```

Comment:
Do you think that there is an indentation error here? Do consider looking through your code for consistency in indentation.